### DISCUSSION TITLE
使用oracle cdc 3.0采集ddl数据，任务运行报错：com.ververica.cdc.connectors.shaded.org.apache.kafka.connect.errors.DataException: file is not a valid field name

### DISCUSSION THREAD

**Discussion by ckh123456** [apache/flink-cdc#3302] [Q&A]
背景：
通过增量快照的方式采集oracle ddl数据
![image](https://github.com/apache/flink-cdc/assets/117636147/7cadc88e-d037-4aa6-a49a-8a35d47a45af)

报错信息如下：
![image](https://github.com/apache/flink-cdc/assets/117636147/41ce0282-2a86-4d54-a4b6-ef18b647c272)

经定位，报错位置如下：
![image](https://github.com/apache/flink-cdc/assets/117636147/fc4c5fed-a315-4067-8f34-c96d89689182)

通过断点查看从debezium传过来的数据貌似并没有file、pos、server_id这些字段。且数据中的字段列表与oracle debezium官网提供的示例是一致的。
![image](https://github.com/apache/flink-cdc/assets/117636147/a70250ff-175a-4c45-851c-12c5a7c3ec50)
![image](https://github.com/apache/flink-cdc/assets/117636147/50c570d7-1f21-428c-b992-7f065f201fa0)

请问，oracle cdc3.0是否支持DDL数据采集？该如何配置？

**Comment by yuxiqian** [comment]
Hi @ckh123456, thanks for reporting this! PR #2315 has fixed schema change event parsing issue, and will be released with version 3.1, which will be released soon. You may also compile the [release candidate version](https://github.com/apache/flink-cdc/tree/release-3.1.0-rc1) manually for any urgent cases.

**Reply by ckh123456** [reply]
Thank you for your reply! Has version 3.1 been officially released?

![image](https://github.com/apache/flink-cdc/assets/117636147/13bba840-515f-45d4-9c9a-b0f08784b11a)

**Reply by yuxiqian** [reply]
Flink CDC maintainers have prepared a release candidate version, and it's still under Apache Software Foundation's voting process. You may track the latest updates [here](https://lists.apache.org/thread/l5p663yw37q5xw633xbc5s1poqbx5n3g) :)

**Reply by ckh123456** [reply]
Is there any other way to collect DDL data from Oracle and SQL Server now? @yuxiqian
